File: TP1/time_example.py
from time import sleep
import time_chrono
import time_profiler


def countdown():
    time_chrono.start()
    for i in range(10):
        sleep(1)
        print("Temps : ", time_chrono.stop())
    print("Temps total écoulé : ", time_chrono.stop())


time_profiler.start()
countdown()
time_profiler.stop()

File: TP1/time_example.c
//
// Created by altaks on 24/11/23.
//

#include "time_example.h"

void countdown(){
    struct timespec startTime;

    start(&startTime);
    for(int i = 0; i < 10; i++){
        bad_sleep(1);
        printf("Temps : %f\n", stop(&startTime));
    }
}
